### PR TITLE
chore: add 3 Paulo Coelho qutoes to german.json (@neinja007)

### PR DESCRIPTION
**Pull request by neinja007**
### Checks

- [x] Adding quotes?
  - [x] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [ ] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [ ] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

Closes #

### CODE REVIEW

File: frontend/static/quotes/german.json
Comment (neinja007) [inline frontend/static/quotes/german.json:1]: Quote 1: **The world is changed by your example, not by your opinion.**
Quote 2: **The only power you have on this planet is the power of your decisions.**
Quote 3: **One day you will wake up & there won’t be any more time to do the things you’ve always wanted. Do it now.**

**Comment by neinja007** [comment]
Here are the translations (also in commit): 

Quote 1: **The world is changed by your example, not by your opinion.**
Quote 2: **The only power you have on this planet is the power of your decisions.**
Quote 3: **One day you will wake up & there won’t be any more time to do the things you’ve always wanted. Do it now.**

**Comment by Miodec** [comment]
For future reference: please don't open a new PR everytime the build fails - you can simply edit the same PR and the checks will rerun automatically.

**Comment by Miodec** [comment]
Thanks

**Comment by neinja007** [comment]
> For future reference: please don't open a new PR everytime the build fails - you can simply edit the same PR and the checks will rerun automatically.

sorry, it was my first contribution. Will it be merged into a release?

**Comment by Miodec** [comment]
Yeah, later today.

**Pull request merged**